Jenkinsfile

```
@@ -51,66 +71,79 @@ try {
-        echo 'Ubuntu Test: Run nodes pool'
+        echo "${env_name} Test: Run nodes pool"
         poolInst = poolEnv.run("--ip=\"10.0.0.2\" --network=${network_name}")
 
-        echo 'Ubuntu Test: Build docker image'
-        def testEnv = dockerHelpers.build(name)
+        echo "${env_name} Test: Build docker image"
+        def testEnv = dockerHelpers.build(name, file)
 
         testEnv.inside("--ip=\"10.0.0.3\" --network=${network_name}") {
-            echo 'Ubuntu Test: Test'
-            sh 'chmod -R 777 /home/indy/'
-            sh 'cargo update'
-
-            try {
-                sh 'RUST_BACKTRACE=1 RUST_TEST_THREADS=1 cargo test --features "interoperability_tests"'
-                /* TODO FIXME restore after xunit will be fixed
-                sh 'RUST_TEST_THREADS=1 cargo test-xunit'
-                 */
-            }
-            finally {
-                /* TODO FIXME restore after xunit will be fixed
-                junit 'test-results.xml'
+           echo "${env_name} Test: Test"
+           sh 'chmod -R 777 /home/indy/'
+           sh 'cargo update'
+
+           try {
+                if (run_interoperability_tests) {
+                    sh 'RUST_BACKTRACE=1 RUST_TEST_THREADS=1 cargo test --features "interoperability_tests"'
+                }
+                else {
+                    sh 'RUST_BACKTRACE=1 RUST_TEST_THREADS=1 cargo test'
+                }
+               /* TODO FIXME restore after xunit will be fixed
+               sh 'RUST_TEST_THREADS=1 cargo test-xunit'
                 */
-            }
+           }
+           finally {
+               /* TODO FIXME restore after xunit will be fixed
+               junit 'test-results.xml'
+               */
+           }
         }
     }
     finally {
-        echo 'Ubuntu Test: Cleanup'
+        echo "${env_name} Test: Cleanup"
         try {
             sh "docker network inspect ${network_name}"
         } catch (ignore) {
```

Remove these catches. You swallow errors here